integration-test/src/main/java/com/sequenceiq/it/cloudbreak/CredentialTests.java

```diff
@@ -79,28 +82,33 @@ public void cleanUp() throws Exception {
         for (String aNameArray : nameArray) {
             LOGGER.info("Delete credential: \'{}\'", aNameArray.toLowerCase().trim());
             try {
-                given(CloudbreakClient.isCreated());
+                given(CloudbreakClient.created());
                 given(Credential.request()
                         .withName(aNameArray + cloudProvider.getPlatform().toLowerCase()));
                 when(Credential.delete());
-            } catch (ForbiddenException e) {
-                String exceptionMessage = e.getResponse().readEntity(String.class);
-                errorMessage = exceptionMessage.substring(exceptionMessage.lastIndexOf(':') + 1);
-                LOGGER.info("ForbiddenException message ::: " + errorMessage);
+            } catch (ForbiddenException forbiddenExp) {
+                try (Response response = forbiddenExp.getResponse()) {
+                    String exceptionMessage = response.readEntity(String.class);
+                    errorMessage = exceptionMessage.substring(exceptionMessage.lastIndexOf(':') + 1);
+                    LOGGER.info("ForbiddenException message ::: " + errorMessage);
+                } catch (WebApplicationException appExp) {
```

Comment:
I suppose this WebApplicationException catch clause is in the wrong place because none of the methods in that specific try block throws this exception, but the API call (Credential.delete()) does (which is a level higher in this nested try-catch situation).